interpreter/Makefile

```diff
@@ -9,131 +9,39 @@
 
 # Configuration
 
-NAME =		wasm
-UNOPT = 	$(NAME).debug
-OPT =   	$(NAME)
-LIB =		$(NAME)
-ZIP =		$(NAME).zip
-JSLIB =		wast
-WINMAKE =	winmake.bat
-
-DIRS =		util syntax binary text valid runtime exec script host main tests
-LIBS =		
-FLAGS = 	-lexflags -ml -cflags '-w +a-4-27-42-44-45-70 -warn-error +a-3'
-OCBA =		ocamlbuild $(FLAGS) $(DIRS:%=-I %)
-OCB =		$(OCBA) $(LIBS:%=-libs %)
-JSO =		js_of_ocaml -q --opt 3
-JS =		# set to JS shell command to run JS tests, empty to skip
+OPT =		wasm.exe
+ZIP =		wasm.zip
+JSLIB =		wast.bc.js
```

Comment:
The end result here should still be named wast.js. May require renaming it in the build target.